uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document.spec.js

```diff
@@ -318,6 +318,122 @@ define([
           expect(DocumentService.save).toHaveBeenCalledWith(jasmine.objectContaining({ activity_date_time: '' }));
         });
       });
+
+      describe('document contains files', function () {
+        beforeEach(function () {
+          angular.extend(controller.document, mockDocument);
+        });
+
+        describe('document already has attachments', function () {
+          beforeEach(function () {
+            controller.files.length = 2;
+          });
+          describe('user is admin', function () {
+            beforeEach(function () {
+              controller.role = 'admin';
+              controller.confirm();
+            });
+
+            it('sets the document status to approved', function () {
```

Comment:
`sets document status to Approved` or `sets document status to "Approved"`, please select one of the suggested or propose your version. The key is to show that "Approved" is a status not just a word. Please fix this in other places in the test.